packages/envision/src/js/dropdown.js

```diff
@@ -75,6 +83,122 @@ const Dropdown = (($) => {
          this.$el.off(`${ENV_KEYDOWN_EVENT}-${this.el.id}`);
       }
 
+      _selectMenuItem(event) {
+         const $items = this.$el.find('.env-dropdown__item');
+
+         if (!$items.length) {
+            return;
+         }
+
+         let index = $items.index(event.target);
+
+         if (event.key === KEYS.ARROWUP && index > 0) {
+            index--;
+         }
+
+         if (event.key === KEYS.ARROWDOWN && index < $items.length - 1) {
+            index++;
+         }
+
+         index = index === -1 ? 0 : index;
+
+         $items[index].focus();
+      }
+
+      static KeydownHandler(event) {
+         if (!REGEXP_KEYDOWN.test(event.key)) {
+            return;
+         }
+
+         const isActive = $(this)
+            .closest(ENV_DROPDOWN)
+            .hasClass(ENV_DROPDOWN_OPEN);
+
+         if (!isActive && event.key === KEYS.ESCAPE_KEY) {
+            return;
+         }
+
+         event.preventDefault();
+         event.stopPropagation();
+
+         const getToggleButton = () =>
+            this.matches(TOGGLE_DROPDOWN)
+               ? this
+               : $(this).closest(ENV_DROPDOWN).find(TOGGLE_DROPDOWN)[0];
+
+         if (event.key === KEYS.ESCAPE_KEY) {
+            Dropdown.clearMenus();
+            getToggleButton().focus();
+
+            return;
+         }
+
+         if (
+            !isActive &&
+            (event.key === KEYS.ARROWUP || event.key === KEYS.ARROWDOWN)
+         ) {
+            getToggleButton().click();
+            return;
+         }
+
+         if (!isActive || event.which === KEYS.SPACE) {
+            Dropdown.clearMenus();
+
+            return;
+         }
+
+         const $this = $(this).closest(ENV_DROPDOWN);
+         const data = $this.data('env.dropdown');
+
+         data._selectMenuItem(event);
```

Comment:
Gör om det här anropet och skicka in `event` och `items`.

```suggestion
         selectMenuItem(event, $dropdown.find('.env-dropdown__item'));
```